pkg/osquery/tables/dataflattentable/tables.go

```diff
@@ -33,7 +42,19 @@ func TablePlugin(client *osquery.ExtensionManagerClient, logger log.Logger) *tab
 		logger: logger,
 	}
 
-	return table.NewPlugin("kolide_plist", columns, t.generate)
+	switch dataSourceType {
+	case PlistType:
+		t.dataFunc = dataflatten.PlistFile
+		t.tableName = "kolide_plist"
+	case JsonType:
+		t.dataFunc = dataflatten.JsonFile
+		t.tableName = "kolide_json"
+	default:
+		panic("Unknown data soure type")
```

Comment:
```suggestion
		panic("Unknown data source type")
```

Reply:
why didn't `misspell` catch this. Anyhow, fixed.